_data/simple-icons.json

```diff
@@ -314,6 +314,11 @@
             "source": "https://www.adyen.com/press-and-media/presskit",
             "guidelines": "https://www.adyen.com/press-and-media/presskit"
         },
+        {
+            "title": "Aegis Authenticator",
+            "hex": "005E9D",
+            "source": "https://alexbakker.me/u/pn8frf828n.svg"
```

Comment:
How is the source at all related to the project?

Reply:
> There's now official vector by [Aegis Authenticator dev](https://github.com/alexbakker): https://alexbakker.me/u/pn8frf828n.svg

From the related issue: Alex Bakker is one of the developers of Aegis which is the only source for the Aegis svg. Unfortunately, the vector is neither hosted in their GitHub repo nor website.

Reply:
I am one of the two developers of Aegis and I can confirm that's our official vector image for Aegis Authenticator. If it helps we can host this image on our website.

Reply:
> I am one of the two developers of Aegis and I can confirm that's our official vector image for Aegis Authenticator. If it helps we can host this image on our website.

I think it is fine but really would be nice if it was a more official URL, the obfuscated filename of `/u/pn8frf828n.svg` certainly doesn't do it any favors 😆

Reply:
@uncenter I agree, you're right! Just uploaded our vector image to our website :relaxed: thanks!

https://getaegis.app/aegis.svg